packages/react-app/src/models/Plot.ts

```diff
@@ -1,15 +1,22 @@
-export interface Plot {
-  id: number;
-  parcel: number;
-  sold: boolean;
-  owner: string;
-  price?: string;
-  metadata: PlotMetadata;
-}
+// export interface Plot {
```

Comment:
You can feel free to remove commented code blocks.
If we need to go back, we can check the commit history or check this branch: https://github.com/citydaoproject/app/tree/old-version-with-minting-feature (I had created a branch before we started working on improvements)